### PR TITLE
Add In-App Messages FAQ

### PR DESCRIPTION
**Pull request by lydia-xie**
# Pull Request/Issue Resolution

**Description of Change:**
> I'm changing..... (could be a link, a new image, a new section, etc.)... because...

Closes #**ISSUE_NUMBER_HERE**

### Is this change associated with a Braze feature/product release?
- [ ] Yes (__Insert Feature Release Date Here__)
- [ ] No

> If yes, please note the date of the feature release.

---
---

<details>
<summary>PR Checklist</summary>

- [ ] Check that all links work!
- [ ] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is _done and ready to be reviewed for merge_.
- [ ] Tag others as Reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

</details>

<details>
<summary>Writing & Style Quick Tips</summary>

- Use the formatting shown on the [Styling Test Page](https://github.com/Appboy/braze-docs/blob/develop/_docs/_home/styling_test_page.md) to create unique templates and vehicles for your information.
- Use Action verbs!
- Capitalize proper nouns, unique Braze tools and features, custom Braze channels, and all Header titles.
- Redirect when necessary!
- Always speak to the Braze customer - refer to them as “you”, whether they are an Engineer/Developer, a Marketer, or an Admin.
- On Voice: You should aim to be a Friendly Professor. Imagine, if you will, the smartest, but also kindest professor you had. You know, the one who showed you your potential and tested you, but also went slowly in class so you could take great notes and learn some things? Don't be too peppy, but don't be formal. You’re not selling anything - you’re instructing and informing and enabling.

_Consult the [Docs Text Formatting Guide](https://github.com/Appboy/braze-docs/wiki/Special-Formatting) and the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/braze-docs/wiki/Writing-Style-Guide-&-Best-Practices) for more details._

</details>

<details>
<summary>ATTENTION: For PR Reviewers</summary>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
</details>

---
---

### CODE REVIEW

File: _docs/_user_guide/message_building_by_channel/in-app_messages/faq.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+tool: In-App Messages
+---
+
+# In-App Messages FAQs
+
+> This article provides answers to some frequently asked questions about in-app messages.
+
+### Will an in-app message display if a device is offline?
+
+It depends. Because in-app messages are delivered at session start, the device is able to download the payload prior to going offline, the in-app message can still be displayed while offline. If the payload is not downloaded, then the in-app message will display.
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/message_building_by_channel/in-app_messages/faq.md:16]: @lydia-xie I think this should say "If the payload is not downloaded, then the in-app message will not display"

File: _docs/_user_guide/message_building_by_channel/in-app_messages/faq.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+### If a user already has an in-app message payload on their device and the message expiration is changed, will the expiration be updated on their device?
+
+When a user starts a session, Braze checks if changes have been made to any in-app messages that they are eligible for and updates them accordingly. So if the expiration has changed and they log a session, then the in-app message is sent to the device with the updated information.
+
+### How do I set up Quiet Hours for an in-app message campaign?
+
+The Quiet Hours feature isn't available for use with in-app message campaigns. This feature is used to prevent messages from being sent to your users during specific hours. For in-app message campaigns, your users will only receive in-app messages if they are active within the app. 
+
+As a workaround to send in-app messages during a specific time, use the following sample Liquid code. This allows the message to be aborted if the in-app message is displayed after 7:59 pm or 8 am at the specified timezone.
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/message_building_by_channel/in-app_messages/faq.md:26]: @lydia-xie I think this should say "after 7:59 pm or before 8 am"

**Review by bre-fitzgerald — CHANGES_REQUESTED** [review]
Looks great! Left a few comments for changes :)

File: _docs/_user_guide/message_building_by_channel/in-app_messages/faq.md (hunk 2 above)
Comment (lydia-xie) [reply _docs/_user_guide/message_building_by_channel/in-app_messages/faq.md]: @bre-fitzgerald Updated based on your suggestion!

File: _docs/_user_guide/message_building_by_channel/in-app_messages/faq.md (hunk 1 above)
Comment (lydia-xie) [reply _docs/_user_guide/message_building_by_channel/in-app_messages/faq.md]: @bre-fitzgerald Updated based on your suggestion!

**Review by bre-fitzgerald — APPROVED** [review]
lgtm!!

**Pull request merged**